script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt

```diff
@@ -415,7 +416,8 @@ But if you're trying to produce vampires of rank II and above,
 [sha:fcaa72b383d7c6afe6a0d5cc0cf8ce3eb3759e4b]{
 -- Page 544, Offset 17931.
 -- 「吸血種の血に抗える免疫力……かな？」
-"Something like an immunity against vampire blood, I guess?"
+// bloodsucker blood :aokoheh:
```

Comment:
i think make this "vampiric blood". "bloodsucker blood" is a little too ridiculous.

Maybe ask petrikow if ross or yusuke disagree

Reply:
I agree